_episodes/04-understanding-yaml-and-ci.md

```diff
@@ -32,6 +32,8 @@ build_code:
 ~~~
 {: .language-yaml}
 
+In the above example, `image: rikorose/gcc-cmake` is a docker image. This image is an extension of the official gcc image and includes cmake.
```

Comment:
Thanks @amanmdesai ! As a picky comment: Docker must have capital 'D'. Could you change it?

Reply:
thanks @michmx I have updated.